src/test/resources/features/tasks/update_task.feature

```diff
@@ -7,7 +7,8 @@ Feature: Update task
       | name          | Updated Task Name    |
     And I execute the PUT request
     Then I verify that the response status is 200
-    And I verify the schema matches the file: schemas/task/update_task_required_field.json
+      And I verify the body matches with response
```

Comment:
the step definition sound better if it says " I verify that the response matches with the body"